components/schema-migrator/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:3.9
```

Comment:
recently @pkosiec added such thing to other Dockerfiles:
```
COPY ./licenses ${BASE_APP_DIR}/licenses
```
please check if we need it also here